numpy/_core/tests/test_cython.py

```diff
@@ -61,9 +61,13 @@ def install_temp(tmpdir_factory):
                               )
     try:
         subprocess.check_call(["meson", "compile", "-vv"], cwd=build_dir)
-    except subprocess.CalledProcessError as p:
-        print(f"{p.stdout=}")
-        print(f"{p.stderr=}")
+    except subprocess.CalledProcessError:
+        print("----------------")
+        print("meson build failed when doing")
+        print(f"'meson setup --native-file {native_file} {srcdir}'")
+        print(f"'meson compile -vv'")
+        print(f"in {build_dir}")
+        print("----------------")
```

Comment:
The stdout/stderr already shows up as part of the error?  The additional info seems nice, but that part is a bit unclear to me.

Reply:
The stdout/stderr reports only that the build fails, and pytest hides everything else. To reproduce locally, it helps to have the cwd and the meson commands.